src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.html

```diff
@@ -361,9 +361,6 @@ <h3>Findings</h3>
           matTooltip="Not guilty as charged but guilty of lesser / included / other offence">
           Not guilty as charged but guilty of lesser / included / other offence
         </mat-option>
-        <mat-option [value]="Finding.PaidPriorToAppearance" matTooltip="Paid Prior to Appearance">Paid Prior to
-          Appearance</mat-option>
-        <mat-option [value]="Finding.Cancelled" matTooltip="Cancelled">Cancelled</mat-option>
       </mat-select>

```

Comment:
I believe this is the correct section to remove these options. This section is only shown when !isSSEditMode. Below is the section for isSSEditMode, when the current user has staff-support permissions and the current screen is staff workbench, not jj workbench. When isSSEditMode==true, then the below section will enable allowing support staff to modify Findings (lines 425-447)

Reply:
Thanks for confirming that Kevin. I have also confirmed this with Sylvia yesterday that the options will be read-only for DCF for regular Staff member and JJs. I think only users who have staff-support permissions will be able to see last two options. I will activate this PR then.